### PR TITLE
[14.0][MIG] web_edit_user_filter migration to 14.0

### PR DESCRIPTION
**Pull request by robyf70**
Migrated to v14

### CODE REVIEW

**Comment by francesco-ooops** [comment]
@robyf70 can you rebase?

**Comment by robyf70** [comment]
> @robyf70 can you rebase?

Done

File: web_edit_user_filter/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Copyright 2019 Onestein
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Edit User Filters",
+    "category": "Extra Tools",
+    "version": "14.0.1.0.1",
```
Comment (i-vyshnevska) [inline web_edit_user_filter/__manifest__.py]:
```suggestion
    "version": "14.0.1.0.0",
```

**Review by i-vyshnevska — APPROVED** [review]
Looks LG

**Comment by HekkiMelody** [comment]
Functional review, LGTM

**Review by HekkiMelody — APPROVED** [review]

**Comment by Highcooley** [comment]
@simahawk does anything speak against merging this migration?

**Review by PicchiSeba — APPROVED** [review]
LGTM

**Comment by grindtildeath** [comment]
@robyf70 Thanks for the mig. There's a small issue where the popover is appearing when we are removing a filter. I did fix it in following PR, can you please merge it? https://github.com/LevelPrime/web/pull/1

**Comment by robyf70** [comment]
> @robyf70 Thanks for the mig. There's a small issue where the popover is appearing when we are removing a filter. I did fix it in following PR, can you please merge it? [LevelPrime#1](https://github.com/LevelPrime/web/pull/1)

Sure!!! I will do right now!

**Comment by robyf70** [comment]
@grindtildeath I've merged everything including a small code review to remove warnings from lint

**Review by grindtildeath — APPROVED** [review]

**Comment by grindtildeath** [comment]
@robyf70 Thank you. But you didn't necessarily need to re request reviews from those who did already approve the PR.

@simahawk Can we merge this with the previous approvals? :pray:

File: web_edit_user_filter/__manifest__.py (hunk 1 above)
Comment (simahawk) [reply web_edit_user_filter/__manifest__.py]: seems still to be addressed

File: web_edit_user_filter/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    "author": "Onestein, Odoo Community Association (OCA),Level Prime Srl",
```
Comment (simahawk) [inline web_edit_user_filter/__manifest__.py]: OCA must be kept at the end

Comment (robyf70) [reply web_edit_user_filter/__manifest__.py]: @simahawk Is it only to keep OCA at the end?

Comment (simahawk) [reply web_edit_user_filter/__manifest__.py]: y

**Comment by robyf70** [comment]
@simahawk Ok! Forgot to change version, now I did it!

**Review by simahawk — APPROVED** [review]

**Pull request merged**